tools/provision/formula/librdkafka.rb

```diff
@@ -0,0 +1,50 @@
+require File.expand_path("../Abstract/abstract-osquery-formula", __FILE__)
+
+class Librdkafka < AbstractOsqueryFormula
+  desc "The Apache Kafka C/C++ library"
+  homepage "https://github.com/edenhill/librdkafka"
+  url "https://github.com/edenhill/librdkafka/archive/v0.9.5.tar.gz"
+  sha256 "dd395ffca89c9591e567366f3ad2517cee76578a10d0a16a93f990c33f553179"
+
+  depends_on "openssl"
+  depends_on "pkg-config" => :build
+  depends_on "lzlib"
+  depends_on "lz4" => :recommended
+
+  def install
+    args = [
+      "--disable-dependency-tracking",
+      "--prefix=#{prefix}",
+    ]
+
+    if OS.mac?
+      args += [
+        "--disable-sasl",
+        "--disable-lz4",
+      ]
+    end
+
+    if OS.linux?
+      ENV.append "LIBS", "-lpthread -lz -lz -lssl -lssl -lcrypto -lcrypto  -lrt"
```

Comment:
Do you need the `-lssl` and `-lcrypto` twice?

Reply:
Not that I know of.  I'll check if it works with just one.  I initially just reordered based on the exact output from the make system call in the build process.